S/SEAL/build_tarballs.jl

```diff
@@ -3,65 +3,107 @@
 using BinaryBuilder, Pkg
 
 name = "SEAL"
-version = v"3.5.6"
+version = v"3.6.2"
 
 # Collection of sources required to complete build
 sources = [
     ArchiveSource("https://github.com/microsoft/SEAL/archive/v$(version).tar.gz",
-                  "13674a39a48c0d1c6ff544521cf10ee539ce1af75c02bfbe093f7621869e3406"),
-    DirectorySource("./bundled")
+                  "1e2a97deb1f5b543640fc37d7b4737cab2a9849f616c13ff40ad3be4cf29fb9c")
 ]
 
 # Bash recipe for building across all platforms
 script = raw"""
 cd $WORKSPACE/srcdir/SEAL-*
 
-# Apply patches that fix some cross-compilation issues
-for f in ${WORKSPACE}/srcdir/patches/*.patch; do
-    atomic_patch -p1 ${f}
-done
-
-# The last three '-DSEAL_USE__*' flags are required to circumvent
+# The last '-DSEAL_USE__*' and `-DSEAL*_EXITCODE*` flags are required to circumvent
 # cross-compilation issues
 if [[ "${target}" == *-darwin* ]]; then
   # C++17 is disabled on MacOS due to the environment being too old.
-  cmake . \
+  cmake -S . -B build \
     -DCMAKE_INSTALL_PREFIX=$prefix \
```

Comment:
This is quite a long list of flags, repeated multiple times.  What about doing something like https://github.com/bramtayl/Yggdrasil/blob/0449a77221ea1628ba8af8be8974233855445e28/L/Libical/build_tarballs.jl#L23-L30 for the common flags and adding the platform-specific flags in the `if`?  That'd be much more maintainable

Reply:
Yes, that makes much more sense. Fixed in b0bdc38a.